packages/cw4/src/helpers.rs

```diff
@@ -79,36 +71,14 @@ impl Cw4Contract {
 
     /// Read the total weight
     pub fn total_weight(&self, querier: &QuerierWrapper) -> StdResult<u64> {
-        let query = self.encode_raw_query(TOTAL_KEY.as_bytes());
-        querier.query(&query)
+        const TOTAL: Item<u64> = Item::new(TOTAL_KEY);
+        TOTAL.query(querier, self.addr())
```

Comment:
Is there a point in creating local `const TOTAL` when similar
`let total = Item::new(TOTAL_KEY);`
is immutable as well? 
Also IMO function names are here quite self-explanatory, so you could go even shorter:
`Item::new(TOTAL_KEY).query(querier, self.addr())`

Reply:
We need to define the type Item returns, so: 

`Item::new::<u64>(TOTAL_KEY).query(querier, self.addr())`

But wait, I guess this is set in the wrapping function signature. Good idea